sirepo/template/activait.py

```diff
@@ -175,6 +175,23 @@ def get_fft_report(run_dir, data):
     return w, plots, f"FFT", summaryData
 
 
+def new_simulation(data, new_sim_data, qcall, **kwargs):
+    if "sourceSimType" not in new_sim_data:
+        return
+    t_basename = f"{new_sim_data.sourceSimType}-{new_sim_data.sourceSimId}-{new_sim_data.sourceSimFile}"
```

Comment:
Remove the temporary. If you feel the need for a temporary, then use `f = data.models.dataFile`.